### PR TITLE
Makes Mining Less Miserable (Details Inside)

### PR DESCRIPTION
**Pull request by PerishedFraud**
## About The Pull Request

Mining was hit very hard by prior PRs in an effort to remove powergaming. In particular, the addition of skeleton keys and removal of all pvp items. It succeeded, but it wouldn't be a lie to say that the coders didn't care about the job experience at all. This PR shuffles some things around in an effort to make mining less of a dry-ass wasteland. It's a good primer before actually adding new mining equipment that isn't powergamey as well, as it removes all the trash.

- Tendrils: Removed worthless drops that made tendrils a monument to miner hate. Down to 11 items: ship, eye, potion, rod, ladder, cubes, wisp, talisman, book, memento and the water bottle.
- Puzzles: Updated drop table accordingly: Paradox bags, ladder, book, water bottle, necropolis seed. Of these, the bags and seed are now exclusive to puzzles.
- Mining Vendor: Now contains accelerator mods previously printable from tendril disks. This is now a viable point sink for miners that might just see actual use. It also makes accelerators more viable compared to crusher. Added: hybrid AoE and repeater for 3000, resonator and syphon for 4000, lifesteal for 5000.
- Crafting: You can now craft a pickaxe, silver pickaxe and diamond pickaxe. Miners likely won't notice, but it's notable for ashwalkers, who'd be hurt by tendril pickaxe removal otherwise.

The hook is gone. If you want to remove powergame from mining, finish the job.

This doesn't make miners more powerful against crew. You can rest easy knowing that.

Please don't remove the PR without any explanation Ike. I know you hate mining but have a heart.

Apologies if the code has mistakes. I'm still shit at it. (Though I did double check).

## Why It's Good For The Game

Makes mining less miserable. Makes accelerators more viable. Adds a point sink. Craftable pickaxes make ashwalkers less miserable too.

## Changelog
:cl:
balance: rebalanced mining drops and vendor.
/:cl:

### CODE REVIEW

**Review by MNarath1 — CHANGES_REQUESTED** [review]
this isn't passing even linter i recommend at the very least try to compile your code if you double check it or at least restart the language server to get problems like this in this case this seems to be mainly caused by the fact you forgot the / before obj in the typepath in line 983, 993 ,1003 in the recipes.dm file making byond think this is a var and as there is no variable like this of course it fails to compile

**Comment by PerishedFraud** [comment]
Thanks frien

**Comment by zeskorion** [comment]
Making the eye more common does make miners better against crew actually.

Either way, lavaland should just be removed and replaced woth asteroid mining

**Comment by PerishedFraud** [comment]
Eye isn't flashproof and putting it on makes you unable to put on shades. It might let you see something suspicious once in a blue moon, but it's very much a mining item and not a pvp one.

File: code/datums/components/crafting/recipes.dm [hunk 1]
```diff
@@ -976,3 +976,32 @@
 	category = CAT_MISC
 	tools = list(TOOL_WIRECUTTER)
 	
+/datum/crafting_recipe/iron_pickaxe
+	name = "Iron Pickaxe"
+	result = /obj/item/pickaxe
+	time = 40
+	reqs = list(/obj/item/stack/sheet/iron = 3,
+			/obj/item/stack/sheet/mineral/wood = 2
```
Comment (PowerfulBacon) [inline code/datums/components/crafting/recipes.dm:984]: Can ashwalkers and stuff even get wood on lavaland?

Comment (PerishedFraud) [reply code/datums/components/crafting/recipes.dm:984]: They have tower caps which give wood. Beyond that, there's always the ore boxes and tables for walkers too lazy for botany.

**Comment by MNarath1** [comment]
> 
> 
> Making the eye more common does make miners better against crew actually.
> 
> Either way, lavaland should just be removed and replaced woth asteroid mining

there would be some potential there if we make something like a hybrit between supercruise asterioids and nsv astroid mining honestly might look into that in the future trough no idea how it would come on for miner mains etc.

**Comment by PerishedFraud** [comment]
The idea was to have lavaland be a destroyed planet, with an "asteroid" cluster that's just bits and pieces of it.

It'd be in space, we'd keep the fauna and some ruins, but a lot would also be replaced. Not to mention split between several areas. Ashwalkers would also be replaced with crashlanded ayys (no tech).

**Comment by Archanial** [comment]
Please test it on local first.

**Comment by PerishedFraud** [comment]
How'd I make it known if I have?

**Comment by PerishedFraud** [comment]
Works on my machine ™

**Comment by PerishedFraud** [comment]
image

**Comment by Fikou** [comment]
port my mining changes coward

**Review by Archanial — APPROVED** [review]

**Comment by Sapfearon** [comment]
add hardsuits and ashdrake transformation back while you at it.
inquisitor hardsuit can be drop from colossus since he has no drop at all aside from voice, khorne hardsuit can be drop from blood drunk miner since it's fitting. they are a bit too strong to be tendril loot, but bosses? i think it's fair.
ashdrake tf was miner holy grail and most fun item you can get which i never, ever saw used in pvp while having 2k hours on bee alone, and god knows how much on tg. It's mostly used to impress people and using it is direct NERF to miner fighting ability in any situation.

**Comment by PerishedFraud** [comment]
What the fuck am I reading

**Comment by BNGP0** [comment]
> Ashwalkers would also be replaced with crashlanded ayys (no tech). 

Well i wanted to give ashwalkers some weak and / or very expensive magic, make space outpost ghost spawns able to do some rituals to spread small amounts of rust on random or chosen parts of the station to scare the crew / call for help and give changelings some connection to magic or add flesh magic fuckery antag and connect these 3 to heretics somehow, which would also take approximately 10 years. So If you really want to turn lavaland into a lava asteroid then keep ashwalkers. Maybe still add crashlanders or maybe even more different ghost spawn roles who can make more ghost spawn roles, it would be cool.

**Review by PerishedFraud — COMMENTED** [review]
it was fixed.

**Pull request closed without merging**